src/contrib/persistence/Akka.Persistence.Sqlite/Journal/BatchingSqliteJournal.cs

```diff
@@ -163,10 +164,7 @@ public BatchingSqliteJournal(Config config) : this(new BatchingSqliteJournalSetu
         {
         }
 
-        /// <summary>
-        /// TBD
-        /// </summary>
-        /// <param name="setup">TBD</param>
+        [Obsolete("Getting removed in v1.6")]
```

Comment:
The batching journals have always been buggy and we're not going to be debugging them anymore. Going to open a separate PR to delete their test suite.